scalyr_agent/log_line_debug.py

```diff
@@ -243,6 +267,29 @@ def serialize(self, output_buffer):
             "Event.serialize", log, line,
         )
 
+class DebugAddEventRequest(scalyr_client.AddEventsRequest, DebugMixin):
+    SEARCH_PATTERN = re.compile(r"threads:\s+(\[.+\]),\s+client_time")
+    def get_payload(self):
+        global CONTAINER_ID
+
+        result = super(DebugAddEventRequest, self).get_payload()
+
+        payload = result.decode("utf-8", "replace")
+
+        if CONTAINER_ID is None:
+            return result
+
+        m = type(self).SEARCH_PATTERN.search(payload)
+        if m:
+            threads_string, = m.groups(1)
+
+            if CONTAINER_ID in threads_string:
+                if file_logger:
+                    log_message(
```

Comment:
I want to be able to quick extract a single addEvents request from the log.  Let's add marker lines before and after you emit this request.  So something like "REQUEST BEGINS" and "REQUEST END" each on newlines by themselves.